os/arch/arm/src/artik05x/src/artik055_i2schar.c

```diff
@@ -78,7 +78,7 @@
 
 static void err_cb(FAR struct i2s_dev_s *dev, FAR void *arg, int flags)
 {
-	lldbg("I2S error callback is engaged with state: 0x%x, arg:  %s\n", flags, arg);
+/* Do something here */
```

Comment:
Add a log to print flags.

Reply:
This is ISR level, not a good place to print things. Needs to do something valuable.